src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -158,6 +159,10 @@ export default {
 			default: () => {},
 		},
 	},
+	async mounted() {
+		await this.$nextTick();
+		addBlankTargetToExternalLinks(this.$refs.heroBodyCopy);
```

Comment:
@michelleinez We'll need to create a prop that can be used to conditionally apply this modification to this page. We don't want this on "all" instances of DynamicHeroClassic, only the instances used in CCLandingPage.